src/components/Clock.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+// eslint-disable-next-line
+import { names } from '../App';
+
+type State = {
+  currentTime: string | null;
+};
+
+type Props = {
+  nameId: number;
+};
+
+export class Clock extends React.Component<Props, State> {
+  state = {
+    currentTime: null,
+  };
+
+  timerId: NodeJS.Timer = setInterval(() => {
+    this.setState({ currentTime: new Date().toLocaleTimeString() });
+    // eslint-disable-next-line
+    console.log(this.state.currentTime);
+  }, 1000);
```

Comment:
Set this interval in componentDidMount, setting intervals like that might be dangerous